### ISSUE TITLE
Train and test set generation 

### ISSUE THREAD

**Issue by chaserileyroberts** [raghavchalapathy/oc-nn#9]
On this line, you're concatenating the test and training sets.
https://github.com/raghavchalapathy/oc-nn/blob/f57130545f221fee67e9780d2a93ca48b9d10ba5/src/models/RCAE.py#L326

Is there a reason for this?

**Comment by raghavchalapathy** [comment]
Since we don’t have labels I had tried experiments with concatenation

With regards
Raghav

**Comment by raghavchalapathy** [comment]
Closing due to inactivity
